### PR TITLE
Clean up DynamicRealmObjectTests

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

File: realm/realm-library/src/androidTest/java/io/realm/DynamicRealmObjectTests.java [hunk 1]
```diff
@@ -557,26 +575,27 @@ public void testUntypedSetUsingStringConversion() {
...
                     case DOUBLE:
                         dObj.set(AllJavaTypes.FIELD_DOUBLE, "1.234");
-                        assertEquals(1.234d, dObj.getDouble(AllJavaTypes.FIELD_DOUBLE));
+                        assertEquals(1.234d, dObj.getDouble(AllJavaTypes.FIELD_DOUBLE), 0f);
                         break;
                     case DATE:
                         dObj.set(AllJavaTypes.FIELD_DATE, "1000");
                         assertEquals(new Date(1000), dObj.getDate(AllJavaTypes.FIELD_DATE));
-
+                        break;
```
Comment (cmelchior) [inline realm/realm-library/src/androidTest/java/io/realm/DynamicRealmObjectTests.java]: Nice catch

Comment (beeender) [inline realm/realm-library/src/androidTest/java/io/realm/DynamicRealmObjectTests.java]: Well, findbugs caught that...

**Comment by cmelchior** [comment]
:+1:

**Comment by dalinaum** [comment]
:+1:

**Pull request merged**